common/script/index.js

```diff
@@ -2281,13 +2282,14 @@ api.wrap = function(user, main) {
           user.stats.lvl++;
           tnl = api.tnl(user.stats.lvl);
           user.stats.hp = 50;
-          if (user.stats.lvl > api.maxLevel) {
+          var userTotalStatPoints = user.stats.str + user.stats.int + user.stats.con + user.stats.per;
+          if (userTotalStatPoints >= api.maxStatPoints) {
             continue;
           }
           if (user.preferences.automaticAllocation) {
             user.fns.autoAllocate();
           } else {
-            user.stats.points = user.stats.lvl - (user.stats.con + user.stats.str + user.stats.per + user.stats.int);
+            user.stats.points = user.stats.lvl - userTotalStatPoints;
```

Comment:
This needs to set user.stats.points to api.maxStatPoints if user.stats.points > api.maxStatPoints. 

user.fns.autoAllocate() should also be checked to ensure it can't increase points above maxStatPoints